Modules/_sqlite/cursor.c

```diff
@@ -310,9 +307,10 @@ _pysqlite_fetch_one_row(pysqlite_Cursor* self)
                     converted = PyUnicode_FromStringAndSize(text, nbytes);
                     if (!converted && PyErr_ExceptionMatches(PyExc_UnicodeDecodeError)) {
                         PyErr_Clear();
-                        colname = sqlite3_column_name(self->statement->st, i);
-                        if (!colname) {
-                            colname = "<unknown column name>";
+                        const char *colname = sqlite3_column_name(self->statement->st, i);
```

Comment:
Adding `const char *` made the line too long. It would better to keep it as it was.

Reply:
I'd prefer this to keep the narrow scoping:
```diff
-                        const char *colname = sqlite3_column_name(self->statement->st, i);
+                        const char *colname;
+                        colname = sqlite3_column_name(self->statement->st, i);
```

Would you be ok with that? Ditto for the other cases.

Reply:
I agree with Serhiy. There is nothing wrong with the previous version. `colname` wasn't used for different purposes inside the function so moving variable definition doesn't make the code more readable as it did in https://github.com/python/cpython/pull/24565.

Reply:
No problem, I'll revert these three cases.